src/main/c/Jep/jep_numpy.c

```diff
@@ -668,12 +652,12 @@ PyObject* convert_jprimitivearray_pyndarray(JNIEnv *env,
 /*
  * Converts a jep.NDArray to a numpy ndarray.
  *
- * @param env    the JNI environment
- * @param obj    the jep.NDArray to convert
+ * @param self   PyJObject representation of th NDArray
+ * @param unused Required by METH_NOARGS
```

Comment:
I don't understand the unused.

Reply:
Because this is used in a [PyMethodDef](https://docs.python.org/3/c-api/structures.html#c.PyMethodDef) it has to be a [PyCFunction](https://docs.python.org/3/c-api/structures.html#c.PyCFunction) which takes 2 args but since the flags indicate it is a [METH_NOARGS](https://docs.python.org/3/c-api/structures.html#METH_NOARGS) the second arg is unused.